### PR TITLE
Delegates and votes

### PR DESCRIPTION
**Pull request by sergushakov**
Refact and cleanup

### CODE REVIEW

File: src/store/modules/delegates/actions.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    })
+  },
+  getForgedByAccount (context, delegate) {
+    admApi.getForgedByAccount(delegate.publicKey).then(response => {
+      if (response.success) {
+        context.commit('update_delegate', {
+          address: delegate.address,
+          params: { forged: response.forged }
+        })
+      } else {
```
Comment (onemorejsdeveloper) [inline src/store/modules/delegates/actions.js]: Empty else block

File: src/store/modules/delegates/actions.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  voteForDelegates (context, payload) {
+    context.commit('clean_delegates')
+    context.commit('ajax_start', null, { root: true })
+    admApi.voteForDelegates(payload.votes).then(response => {
+      if (response.success) {
+        context.commit('set_last_transaction_status', false)
+        // removing an UI waiting state if transaction confirmation run to much time
+        window.setTimeout(() => {
+          if (!context.lastTransactionConfirmed) {
+            console.log('transaction confirmation awaiting')
```
Comment (onemorejsdeveloper) [inline src/store/modules/delegates/actions.js]: Just console.log()?

File: src/store/modules/delegates/actions.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        context.dispatch('getDelegates', { address: payload.address })
+      }
+    })
+  },
+  getForgingTimeForDelegate (context, delegate) {
+    admApi.getNextForgers().then(response => {
+      if (response.success) {
+        const nextForgers = response.delegates
+        const fIndex = nextForgers.indexOf(delegate.publicKey)
+        const forgingTime = fIndex === -1 ? -1 : fIndex * 10
```
Comment (onemorejsdeveloper) [inline src/store/modules/delegates/actions.js]: Possible to make as `fIndex === -1 || fIndex * 10`

File: src/store/modules/delegates/actions.js (hunk 1 above)
Comment (sergushakov) [reply src/store/modules/delegates/actions.js]: fxd

File: src/store/modules/delegates/actions.js (hunk 2 above)
Comment (sergushakov) [reply src/store/modules/delegates/actions.js]: it's debug block, fxd

**Pull request merged**